src/work-allocation-2/models/cases/case-field-config.model.ts

```diff
@@ -0,0 +1,11 @@
+import { FieldType } from '../../enums';
+
+export default interface CaseFieldConfig {
```

Comment:
_Question:_
Is this identical to `TaskFieldConfig`? Is there scope for a more generic `TaskFieldConfig`?

Reply:
I honestly tried doing this however after making changes to 11 files I searched for TaskServiceConfig and it would involve changing a significant number of files. Suggest re-addressing this at a later date (IP week?)